src/StatusCode/Vendor/Apache.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Interface representing extended HTTP status codes for Apache. These codes are
+ * represented as an interface so that developers may implement it and then use
+ * parent::[CODE] to gain a code, or to extend the codes using static::[CODE]
+ * and override their default description.
+ *
+ * This allows for codes to be repurposed in a natural way where the core,
+ * traditional use would not be meaningful.
+ *
+ * PHP version 5.3
+ *
+ * @category   StatusCode
+ * @package    Teapot
+ * @subpackage HttpResponse
+ * @author     Barney Hanlon <barney@shrikeh.net>
+ * @copyright  2013 B Hanlon. All rights reserved.
+ * @license    MIT http://opensource.org/licenses/MIT
+ * @link       http://shrikeh.github.com/teapot
+ */
+namespace Teapot\StatusCode\Vendor;
+
+/**
+ * Interface representing extended HTTP status codes for Apache. These codes are
+ * represented as an interface so that developers may implement it and then use
+ * parent::[CODE] to gain a code, or to extend the codes using static::[CODE]
+ * and override their default description.
+ *
+ * This allows for codes to be repurposed in a natural way where the core,
+ * traditional use would not be meaningful.
+ *
+ * @category   StatusCode
+ * @package    Teapot
+ * @subpackage HttpResponse
+ * @author     Barney Hanlon <barney@shrikeh.net>
+ * @copyright  2013 B Hanlon. All rights reserved.
+ * @license    MIT http://opensource.org/licenses/MIT
+ * @link       http://shrikeh.github.com/teapot
+ */
+
+interface Apache
+{
+    /**
+     * This status code, while used by many servers, is not specified in any RFCs.
+     * Apache-specific.
+     *
+     * @var integer
```

Comment:
@localheinz  I use @var for constants, simply because I can find no alternative in the phpDoc documentation, and that it also aids IDEs (one of the principle reasons for such verbose descriptions in the library).